src/views/v2/modal/target.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Modal: Target
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/modal/target.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1amp
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+?>
+
+<div class="tribe-common event-tickets">
+	<span id="tribe-tickets__modal_target"></span>
```

Comment:
Inherited: https://github.com/moderntribe/event-tickets/blob/ab3073eb15cc67725292e18169c7cc0dc7e6ba83/src/views/blocks/tickets/submit-button-modal.php#L56

Looks like it's the place where the modal content will be added.

Reply:
Correct, content will be added in this spot.

@bordoni Do you think that it might be beneficial to maybe add an HTML or PHP comment here or should we leave as is?